raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -856,6 +856,38 @@ def test_handle_offchain_secretreveal():
     })
 
 
+def test_handle_offchain_emptyhash_secret():
+    channel1 = factories.make_channel(
+        our_balance=UNIT_TRANSFER_AMOUNT,
+        token_address=UNIT_TOKEN_ADDRESS,
+        token_network_identifier=UNIT_TOKEN_NETWORK_ADDRESS,
+    )
+    channel_map = {channel1.identifier: channel1}
+    available_routes = [factories.route_from_channel(channel1)]
+    pseudo_random_generator = random.Random()
+    block_number = 10
+
+    manager_state = make_initiator_manager_state(
+        available_routes,
+        factories.UNIT_TRANSFER_DESCRIPTION,
+        channel_map,
+        pseudo_random_generator,
+        block_number,
+    )
+
+    secret_reveal = ReceiveSecretReveal(
+        secret=EMPTY_HASH,
+        sender=channel1.partner_state.address,
+    )
+    iteration = initiator.handle_offchain_secretreveal(
+        initiator_state=manager_state.initiator,
+        state_change=secret_reveal,
+        channel_state=channel1,
+        pseudo_random_generator=pseudo_random_generator,
+    )
+    assert len(iteration.events) == 0
```

Comment:
Check that the lock hasn't been moved here

Reply:
will do